model/internal/pdata/generated_metrics.go

```diff
@@ -1430,12 +1430,18 @@ func (ms HistogramDataPoint) SetCount(v uint64) {
 
 // Sum returns the sum associated with this HistogramDataPoint.
 func (ms HistogramDataPoint) Sum() float64 {
-	return (*ms.orig).Sum
+	return (*ms.orig).GetSum()
```

Comment:
nit: separate PR we should update this comment that a "zero-initialized" value is returned if the value is not present (HasSum() == false)